### DISCUSSION TITLE
Server Error: Reverse for 'moduletype_add' not found.

### DISCUSSION THREAD

**Discussion by whizkidTRW** [netbox-community/netbox#11201] [Q&A]
Anyone ever seen this? We were prepping for an incremental upgrade from 3.1.4 to 3.1.8 in preparation for going on to 3.3.10 but we keep getting this error randomly:

<class 'django.urls.exceptions.NoReverseMatch'>

Reverse for 'moduletype_add' not found. 'moduletype_add' is not a valid view function or pattern name.

Python version: 3.8.10
NetBox version: 3.1.8

**Comment by abhi1693** [comment]
I have not seen this error either with 3.3.10 or 3.4.0. Maybe try upgrading to newer releases because bug fixes for older versions is never rolled out

**Comment by whizkidTRW** [comment]
After upgrading, the error went away. Still not sure of what the problem was.
